spec/support/vcr_cassettes/lighthouse/benefits_discovery/200_response_without_params.yml

```diff
@@ -0,0 +1,77 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://sandbox.lighthouse.va.gov/benefits-discovery-service/v0/recommendations
```

Comment:
Please mask the URI in [support/vcr](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr.rb)